app/controllers/restaurants_controller.rb

```diff
@@ -10,5 +10,6 @@ def index
 
   def show
     @restaurant = Restaurant.find(params[:id])
+    @menu = Menu.all
```

Comment:
Yes. Why are you taking all menus in the DB when you're showing a particular restaurant? Also if it's more than 1 menu, then we should probably call it `@menus`